src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AesGcm.macOS.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Win32.SafeHandles;
+
+namespace System.Security.Cryptography
+{
+    public sealed partial class AesGcm
+    {
+        private byte[]? _key;
+
+        // CryptoKit added AES.GCM in macOS 10.15, which is our minimum target for macOS.
+        public static bool IsSupported => true;
+
+        // CryptoKit only supports 16 byte tags.
+        public static KeySizes TagByteSizes { get; } = new KeySizes(16, 16, 1);
+
+
```

Comment:
Double blank line

```suggestion
```

Reply:
Ah, so that's where the missing newline from your comment above went.